src/main/java/com/lohika/morning/ecs/domain/template/Template.java

```diff
@@ -0,0 +1,52 @@
+package com.lohika.morning.ecs.domain.template;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "templates")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class Template {
```

Comment:
Let's rename Template to CampaignTemplate, because we will also have EmailTemplate